extism_pdk.go

```diff
@@ -359,80 +410,87 @@ func (r *HTTPRequest) Send() HTTPResponse {
 	}
 }
 
+// `ReadBytes` returns the host memory block as a slice of bytes.
```

Comment:
I don't know that these backticks should be included here, from what I can tell the function name is typically left unstyled?
```suggestion
// ReadBytes returns the host memory block as a slice of bytes.
```

Reply:
Yes, I think you are right... I think I over-zealously back-ticked.  I'll remove them all from the function names and push the changes.